HousingRepairsOnlineApi/UseCases/RetrieveAddressesUseCase.cs

```diff
@@ -26,16 +27,27 @@ public async Task<IEnumerable<Address>> Execute(string postcode)
             if (!string.IsNullOrEmpty(postcode))
             {
                 var addresses = await addressGateway.Search(postcode);
-                result.AddRange(addresses.Select(address => new Address()
+                result.AddRange(addresses.Select(ConvertAddresses));
+            }
+
+            return result;
+
+            Address ConvertAddresses(PropertyAddress address)
```

Comment:
This is more of a nit-picky suggestion but can we name this something more descriptive like ConvertToHactProperyAddress

Reply:
I've renamed the method.